dynamicio/config.py

```diff
@@ -199,6 +199,7 @@ def get(self, source_key: str) -> Mapping:
         source_config = self.config[source_key][self.env_identifier]
         if self.config[source_key].get("schema"):
             schema_definition = self._get_schema_definition(source_key)
+            source_config["name"] = schema_definition["name"]
```

Comment:
Nice, I take it this is used to identify the `table_name` correctly. 👍🏼

Reply:
yeah exactly.